src/components/cluster/detail/cluster_detail_table.js

```diff
@@ -15,18 +16,14 @@ class ClusterDetailTable extends React.Component {
     modifiedPropsPaths: [],
   };
 
-  componentDidMount = () => {
+  componentDidMount() {
     this.registerRefreshInterval();
-  };
-
-  componentWillUnmount = () => {
-    window.clearInterval(this.reRenderInterval);
-  };
+  }
```

Comment:
I looked up a bit on the meaning of the arrow notation for a class method. It wasn't approprioate to apply that here, so I am reverting it. It's unrelated.